src/pages/search/index.tsx

```diff
@@ -128,32 +128,31 @@ const SearchPage: NextPage = () => {
   };
 
   // search facet handlers
-  const [showFilters, setShowFilters] = useState(true);
-  const handleToggleFilters = () => setShowFilters(not);
+  const showFilters = useStore((state) => state.settings.searchFacets.open);
+  const handleToggleFilters = useStore((state) => state.toggleSearchFacetsOpen);
   const handleSearchFacetSubmission = (queryUpdates: Partial<IADSApiSearchParams>) => {
     const search = makeSearchParams({ ...params, ...queryUpdates, p: 1 });
     void router.push({ pathname: router.pathname, search }, null, { scroll: false, shallow: true });
   };
 
   return (
     <>
-      {/* <Box ref={showFiltersBtnRef}></Box> */}
+      <Head>
+        <title>{params.q} | NASA Science Explorer - Search Results</title>
+      </Head>
       <Grid
         aria-labelledby="search-form-title"
         my={12}
         templateColumns={showFilters ? `200px 1fr` : `1rem 1fr`}
         gap="2"
       >
-        <Head>
-          <title>{params.q} | NASA Science Explorer - Search Results</title>
-        </Head>
         {showFilters ? (
           <GridItem as="aside" aria-labelledby="search-facets">
             <Flex>
               <Heading as="h2" id="search-facets" fontSize="sm" flex="1">
                 Filters
               </Heading>
-              <Button variant="link" onClick={handleToggleFilters}>
+              <Button variant="link" onClick={() => handleToggleFilters()}>
```

Comment:
Just curious why this was changed? Does `onClick={() => handleToggleFilters()}` not work here?

Reply:
not sure, but probably I had added some other call there and then removed... I'll fix!